packages/api/src/controllers/userFarmController.js

```diff
@@ -500,7 +500,7 @@ const userFarmController = {
   patchPseudoUserEmail() {
```

Comment:
Refactor this name to reflect that not only the email is being patched. Maybe `upgradePseudoUser` or something.